index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Romanian Esports League 2022</title>
+    <script src="menu.js" defer></script>
+    <script src="dynamic.js" defer></script>
+    <script src="https://kit.fontawesome.com/ca98068cf1.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="displayNoneSmall">
+        <ul class="flex listNone headerItems">
+            <li><a href="#"><i class="fa-brands fa-facebook-f white"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-twitter white"></i></a></li>
+            <li><a href="#" class="white">English</a></li>
+            <li><a href="#" class="white">My page</a></li>
+            <li><a href="#" class="white">Logout</a></li>
+        </ul>
+    </header>
+    <nav class="flex justifySpaceBetween flexAlignCenter">
+        <a href="index.html" class="navLogo displayNoneSmall"><img  src="./img/RSL.png" alt="logo"></a>
+        <i class="displayHideBig hamburger fa-sharp fa-solid fa-bars fa-2x"></i>
+        <ul class="listNone menuList">
+            <li class="closeButton displayHideBig"><i class="fa-solid fa-xmark"></i></li>
+            <li><a class="homeButton displayNone" href="index.html">Home</a></li>
+            <li><a class="primary" href="./about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li class="menuBorder"><a class="primary" href="#">REL Campaign</a></li>
+        </ul>
+    </nav>
```

Comment:
- [ ] **[OPTIONAL]**  Kindly correct: the space between the elements in the navbar and the margin to look similar to the original design 👇🏽 

> - You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including Layout: composition, and space between elements.

![image](https://user-images.githubusercontent.com/58642949/190375726-74d901bb-59a5-4d02-ac05-6981cb162d65.png)